inception-doc/src/main/resources/META-INF/asciidoc/user-guide/workload_settings.adoc

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020
+// Ubiquitous Knowledge Processing (UKP) Lab
+// Technische Universität Darmstadt
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[sect_workload_settings]]
+= Workload settings
+
+NOTE: This functionality is only available when the experimental feature dynamic-workload is enabled.
+
+This configuration tab for a project enables the utilization of the dynamic workload feature. The dropbox lets
+the project manager choose, whether the standard monitoring or the dynamic workload feature shall be used.
+There can only be one of them in use at any given time.
+Upon selecting the dynamic workload from the dropdown and confirming it with the corresponding button, the
+project manager is able to select the new workload tab from the projects menu bar.
```

Comment:
Add how this will also impact the annotators worklfow on the annotation page, please.